cypress/e2e/shared/deepLinks.test.ts

```diff
@@ -12,127 +12,111 @@ describe('Deep linking', () => {
   // so you'll probably need to follow-up with the docs and/or marketing teams.
```

Comment:
it might be helpful to view this diff with whitespace changes off: https://github.com/influxdata/ui/pull/5094/files?w=1